Sources/VitaminUIKit/Components/Snackbar/UIViewController+VitaminSnackbar.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Vitamin iOS
+//  Apache License 2.0
+//
+
+import UIKit
+
+/// Extension that adds the ability to aby `UIViewController` to disply a `VitaminSnackbar` at the bottom of its view
+public extension UIViewController {
+    private static let horizontalPadding = 10.0
+    static let snackbarVerticalPadding = 10.0
+
+    /// Method that will push a `VitaminSnackbar`at the bottom of the controller's view.
+    /// Every property of the snackbar is settable
+    /// - Parameters:
+    ///   - title: title of the `VitaminSnackbar`. Mandatory
+    ///   - content: content of the `VitaminSnackbar`. Mandatory
+    ///   - image: optional image of the `VitaminSnackbar`. `nil` by default
+    ///   - autoDismiss: indicates if the `VitaminSnackbar` should be automatically dismmissed after a delay. Default value is `true`
+    ///   - dismissDelay: delay after which the `VitaminSnackbar`is dimissed automatically. only used if autodimiss is `true`
+    ///   - dismissOnTap: indicates if the `VitaminSnackbar`should be dismissed when user taps on it. Default value id `true`
+    ///   - bottomMargin: space betwwen the bottom of the view and the bottom of the `VitaminSnackbar`. Default value is 10
+    ///   - maxWidth: optional max width of the `VitaminSnackbar`. It set to `nil`, width of the view minus 10 on each side will e applied.
+    ///   default value id `nil`
+    ///
+    func pushVitaminSnackbar(
```

Comment:
I'm not a huge fan of `pushVitaminSnackbar` because we don't push anything. I think that it could be better to use `presentVitaminSnackbar` or just `present(_ snackbar: VitaminSnackbar)` to fit UIKit API.

Reply:
Ok, changed.
But I had to keep two extra arguments : 
- maxWidth
- bottomMargin